test/suites/smoke-test/test-block-finalized.ts

```diff
@@ -0,0 +1,31 @@
+import { describeSuite, expect, beforeAll } from "@moonwall/cli";
+import { getBlockTime } from "@moonwall/util";
+
+import { ApiPromise } from "@polkadot/api";
+const debug = require("debug")("smoke:block-finalized");
```

Comment:
you dont need to do this, you can just use the `log` function which is provided below on line 11:
```
testCases: ({ it, context }) => {
```